src/server/shell/basic_accessibility_manager.h

```diff
@@ -17,10 +17,12 @@
 #ifndef MIR_SHELL_BASIC_ACCESSIBILITY_MANAGER_H
 #define MIR_SHELL_BASIC_ACCESSIBILITY_MANAGER_H
 
+#include "mir/input/input_event_transformer.h"
 #include "mir/shell/accessibility_manager.h"
 
 #include "mir/input/mousekeys_keymap.h"
 #include "mir/synchronised.h"
+#include <cassert>
```

Comment:
Do not include `cassert` in a header file: there should be nothing in a header that uses it: if there is, it is very easy to break the One Definition Rule by accident: two translation units can include the header with different values for `NDEBUG`.